deployment/tf/modules/v2/allocator.tf

```diff
@@ -244,6 +244,6 @@ resource "kubernetes_service" "allocator_service" {
       port = local.allocator-port
       target_port = local.allocator-port
     }
-    type = "LoadBalancer"
+    type = "ClusterIP"
```

Comment:
We need LoadBalancer here to access allocator from workers

Reply:
Yeah, it's just my mistake that I accidentally committed these changes into this branch. It was for testing purpose only. I'll get rid of them